### DISCUSSION TITLE
Icons for Segment

### DISCUSSION THREAD

**Discussion by brannytempy** [twilio-labs/paste#2840] [Request: Icon] [answered]
Hello Paste team!

I have completed making Segment's icon selects and edits. This went through:

1. Segment Designers for Feedback
2. Paste Review Channel

I've made a zip file that contains all of the cleaned SVGs. (See [ZIP of SVGs in Drive](https://drive.google.com/file/d/1r7QCV9Bdww3TrckqfH22GEg4HE6GUxOl/view?usp=share_link))

[Here is a link to the Figma Branch](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/branch/1Tz1QElYz96z20j81695tk/Paste-Components?node-id=124%3A0&t=uOcEXg7RLHkjY9i2-1)

Please let me know of next steps!! 🥳

**Comment by SiTaggart** [comment]
Hi @brannytempy 

Looks like you are up to the ["Exporting the Icon"](https://paste.twilio.design/introduction/contributing/icons/#exporting-the-icon) part of the process.

Next step is to partner with one of the Engineers on the DSYS team to ["Add the icons to the Repository"](https://paste.twilio.design/introduction/contributing/icons/#adding-the-icon-to-the-paste-repository), it's not a super long task, mainly copy paste, so you can grab 30 minutes of someones time. I give you permission to bug someone 😉 .

Then get that Figma branch reviewed by @twilio-labs/design-systems-pd and get it merged.

**Comment by serifluous** [comment, marked as answer]
Icons have been added in #2894
